game_api.txt

```diff
@@ -103,6 +103,14 @@ The contents of `creative.formspec_add` is appended to every creative inventory
 page. Mods can use it to add additional formspec elements onto the default
 creative inventory formspec to be drawn after each update.
 
+Group overrides can be used for any registered item/node. Use one or multiple
```

Comment:
[...] for any registered item, node or tool

Reply:
Done!